src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3915,13 +3915,16 @@ private void CreateInstanceCheckThis()
                 }
 
                 MethodBase? invokeMethod;
-                object? state = null;
+                object? state;
 
                 try
                 {
                     invokeMethod = binder.BindToMethod(bindingAttr, cons, ref args, null, culture, null, out state);
                 }
-                catch (MissingMethodException) { invokeMethod = null; }
+                catch (MissingMethodException mme) // Should we catch all exceptions?
```

Comment:
We should not catch all exceptions here.

Reply:
Thank you for replying. I updated the comment on the code.